src/main/java/uk/gov/ons/census/action/model/repository/FulfilmentsToSendRepository.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.census.action.model.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RestResource;
+import uk.gov.ons.census.action.model.entity.FulfilmentsToSend;
+
+@RestResource(exported = false)
+public interface FulfilmentsToSendRepository extends JpaRepository<FulfilmentsToSend, String> {
+  @Query(
+      value = "SELECT * FROM actionv2.fulfilments_to_send where fulfilment_code = :fulfilmentCode",
+      nativeQuery = true)
+  FulfilmentsToSend findByFulfilmentCode(@Param("fulfilmentCode") String fulfilmentCode);
```

Comment:
you could literally just write `FulfilmentsToSend findOneByFulfilmentCode(String fulfilmentCode);` and not have the `@Query` part at all - JPA will write that for you

Reply:
actually, this is only used by a test and should be deleted - we don't want test code mixed in with the production code